templates/trackingcode.html

```diff
@@ -0,0 +1,19 @@
+{#
+# TODO: create the piwik project and set the PIWIK_ID
+-#}
+
+{%- if config.PIWIK_ID %}
+<script type="text/javascript">
+var _paq = _paq || [];
+_paq.push(["setDomains", ["invenio-software.org"]]);
```

Comment:
Note that canonical URL is without dash, i.e. `inveniosoftware.org`.  Also, we may perhaps need `www.inveniosoftware.org` subdomain in front due to top-level domain name CNAME restrictions... to be seen.